packages/lit-dev-content/site/docs/components/events.md

```diff
@@ -46,7 +46,7 @@ If you're not using decorators, you can customize event listener options by pass
 
 ```js
 render() {
-  return html`<button @click=${{handleEvent: () => this.onClick(), once: true}}>click</button>`
+  return html`<button @click=${{handleEvent: (() => this.onClick()), once: true}}>click</button>`
```

Comment:
Why do you need the outer parens? I thought the previous syntax was valid.

Reply:
errr i think i meant to wrap `this.onClick()` in parens. 

Attempting to figure out this error

```
[11ty] 1. Having trouble rendering liquid template ./site/docs/components/events.md (via TemplateContentRenderError)
[11ty] 2. unexpected token at ": () => this....", file:./site/docs/components/events.md, line:40, col:31 (via ParseError)
[11ty] 3. unexpected token at ": () => this...." (via AssertionError)
[11ty] 
[11ty] Original error stack trace: AssertionError: unexpected token at ": () => this...."
```

Reply:
Cant figure it out rn and it's friday. Just going to revert the PR